src/coreclr/jit/emitarm64.h

```diff
@@ -710,6 +719,12 @@ static bool isValidUimm3(ssize_t value)
     return (0 <= value) && (value <= 7);
 };
 
+// Returns true if 'value' is a legal unsigned immediate 3 bit encoding, starting from 1 (such as for SHRNB).
+static bool isValidUimm3From1(ssize_t value)
+{
+    return (1 <= value) && (value <= 8);
```

Comment:
some of the numbers in `isValid*` are hex while others are decimal, do you mind changing to hex wherever it makes sense?

Reply:
Sure thing. By "wherever it makes sense," do you mean we should only convert decimal numbers to hex if they're >=10? Because in the above example, there wouldn't be anything to change.

Reply:
yeah it was a nit comment, not for this particular example, but there are examples of ` <= 15` that can be converted to ` <= 0xF`  but other places like `(value <= 224) ` can stay decimals.